src/core/parser/compiled-parser.js

```diff
@@ -0,0 +1,343 @@
+var o=function(k,v,o,l){for(o=o||{},l=k.length;l--;o[k[l]]=v);return o},$V0=[1,16],$V1=[1,17],$V2=[1,12],$V3=[1,19],$V4=[1,18],$V5=[5,18,19,23,34,36,77],$V6=[1,23],$V7=[1,24],$V8=[1,26],$V9=[1,27],$Va=[5,14,16,18,19,21,23,34,36,77],$Vb=[1,30],$Vc=[1,34],$Vd=[1,35],$Ve=[1,36],$Vf=[1,37],$Vg=[5,14,16,18,19,21,23,26,34,36,77],$Vh=[1,50],$Vi=[1,49],$Vj=[1,44],$Vk=[1,45],$Vl=[1,46],$Vm=[1,51],$Vn=[1,52],$Vo=[1,53],$Vp=[1,54],$Vq=[1,55],$Vr=[5,16,18,19,23,34,36,77],$Vs=[1,71],$Vt=[1,72],$Vu=[1,73],$Vv=[1,74],$Vw=[1,75],$Vx=[1,76],$Vy=[1,77],$Vz=[1,78],$VA=[1,79],$VB=[1,80],$VC=[1,81],$VD=[1,82],$VE=[1,83],$VF=[1,84],$VG=[1,85],$VH=[26,46,51,53,54,55,56,57,58,60,61,62,63,64,65,66,67,68,70,78],$VI=[26,46,51,53,54,55,56,57,60,61,62,63,64,65,66,67,68,70,78],$VJ=[26,46,51,70,78],$VK=[1,122],$VL=[1,123],$VM=[26,46,51,53,54,60,61,62,63,64,65,66,67,68,70,78],$VN=[26,46,51,60,61,62,63,64,65,66,67,68,70,78],$VO=[51,70],$VP=[16,18,19,23,34,77];
```

Comment:
compiled code, scan for anything harmful.

Compiling the parser ahead of time means we can avoid bundling jison and cuts the minified file size from 300+ kb to 44kb.